.ruby-version

```diff
@@ -1 +1 @@
-2.1.1
+2.5.3
```

Comment:
Should this Ruby version upgrade be in here too?

Reply:
Does it matter?

Reply:
i.e. what's the bad stuff that changing it might cause?

Reply:
🤷‍♂ I mean I guess it could cause things not work locally but really it was more a matter of it having nothing to do with what the PR description and commit messages say this PR is doing so it looked like it wasn't meant to be included.

Reply:
Oh, I had to update the versions in the `.travis.yml` to get the build green, so it seemed like a good time to bump the version in `.ruby-version` to match $LATEST from `.travis.yml`.